service/src/main/java/bio/terra/cbas/util/methods/GithubUrlDetailsManager.java

```diff
@@ -5,7 +5,7 @@
 @Component
 public class GithubUrlDetailsManager {
 
-  public GithubUrlComponents extractDetailsFromUrl(String url) {
+  public static GithubUrlComponents extractDetailsFromUrl(String url) {
```

Comment:
I made this static because it doesn't actually use any class properties - let's talk if you think this is not a good idea for some reason